inc/config.class.php

```diff
@@ -342,6 +350,21 @@ function showForm ($ID, $options = []) {
       } else {
          echo "<input type='hidden' name='warranty_duration' value='0'>\n";
       }
+	// MODIF HP
+      if ($this->fields["name"] == self::HP) {
+         echo "<tr>";
+         echo "<td class='tab_bg_2 center' colspan='2'>".__('Manufacturer API key', 'manufacturersimports')."</td>";
+         echo "<td class='tab_bg_2 left' colspan='2'>";
+         echo "<input type='text' name='supplier_key' size='32' value='".$this->fields["supplier_key"]."'>";
```

Comment:
Increase the size to 100 to be sure in the future not have problem if key more long